packages/app_ui/app_gallery/lib/screens/widget_screens/app_bottom_nav_screen.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:app_ui/app_ui.dart';
+import 'package:flutter/material.dart';
+
+class AppBottomNavigationBarScreen extends StatefulWidget {
```

Comment:
Usually, it's nice to have the same name of the file as the name of the widget.

In this particular case, the file should be named "app_bottom_navigation_bar_screen.dart".